Misc/NEWS.d/next/Build/2021-12-12-05-12-40.bpo-23469.0CypQi.rst

```diff
@@ -0,0 +1 @@
+Delete Wing IDE project files from Misc. Patch by Kumar Aditya.
```

Comment:
Please mention that the Wing IDE is no longer maintained.

Reply:
These files are development-only and probably used by just one person. I don't think they warrant a NEWS entry.